tests/testthat/test-shinytest.R

```diff
@@ -0,0 +1,275 @@
+testthat::test_that(
+  "e2e: teal.widgets::verbatim_popup is initialized with a button that opens a modal with a verbatim text",
+  {
+    skip_if_too_deep(5)
+    module_label <- "Module with verbatim popup"
+    ui_popup_button_label <- "Open popup"
+    modal_title <- "My custom title"
+    verbatim_content_text <- "if (TRUE) { print('Popups are the best') }"
+
+    app <- TealAppDriver$new(
```

Comment:
teal.widgets is independent on teal - it should be using normal appDriver instead of TealAppDriver. We should test teal.widgets in normal shiny application and not in teal specifically.